Robot (screen) freeze

When a chunk is unloaded (I logged out from a server to test this) any powered up robots get their screens frozen. Robot still powered on, last message is readable on screen, no blinking cursor. Upon first key press the HDD/disk sounds shortly, then nothing. The robot must be turned off and on again so that keyboard input is available again.
Did additional testing with Forge 2254, where no native lua libs are available. With this setup the robots are simply turned off, screens black.
Computers and computer screens work fine.

Setup:
Mods: Tested with OpenComputers only
OC 1.17.10 - 1.6.2.12 universal, Forge 1614
OC 1.8.9 - 1.6.2.12 universal, Forge 1902
OC 1.10.2 - 1.6.2.7, Forge 2185

Tested again with disablePersistence=true (OC 1.10.2 - 1.6.2.7, Forge 2185), got this error while reloading the chunk:

full log: http://pastebin.com/nKCpx9tg

[Server thread/ERROR] [OpenComputers/]: Unexpected error loading a state of computer at (684.5, 80.5, 109.5). State: Paused. Unless you're upgrading/downgrading across a major version, please report this! Thank you.
java.lang.Exception: li.cil.repack.com.naef.jnlua.LuaRuntimeException: Invalid kernel.
	at li.cil.oc.server.machine.luac.NativeLuaArchitecture.load(NativeLuaArchitecture.scala:351) ~[NativeLuaArchitecture.class:?]
	at li.cil.oc.server.machine.Machine.liftedTree1$1(Machine.scala:740) [Machine.class:?]
	at li.cil.oc.server.machine.Machine.load(Machine.scala:739) [Machine.class:?]
	at li.cil.oc.common.tileentity.traits.Computer$class.readFromNBTForServer(Computer.scala:151) [Computer$class.class:?]
	at li.cil.oc.common.tileentity.RobotProxy.li$cil$oc$common$tileentity$traits$RotatableTile$$super$readFromNBTForServer(RobotProxy.scala:26) [RobotProxy.class:?]
	at li.cil.oc.common.tileentity.traits.RotatableTile$class.readFromNBTForServer(RotatableTile.scala:38) [RotatableTile$class.class:?]
	at li.cil.oc.common.tileentity.RobotProxy.readFromNBTForServer(RobotProxy.scala:137) [RobotProxy.class:?]
	at li.cil.oc.common.tileentity.traits.TileEntity$class.readFromNBT(TileEntity.scala:92) [TileEntity$class.class:?]
	at li.cil.oc.common.tileentity.RobotProxy.func_145839_a(RobotProxy.scala:26) [RobotProxy.class:?]
	at net.minecraft.tileentity.TileEntity.func_190200_a(TileEntity.java:123) [aqk.class:?]
	at net.minecraft.world.chunk.storage.AnvilChunkLoader.loadEntities(AnvilChunkLoader.java:486) [atj.class:?]
	at net.minecraftforge.common.chunkio.ChunkIOProvider.syncCallback(ChunkIOProvider.java:96) [ChunkIOProvider.class:?]
	at net.minecraftforge.common.chunkio.ChunkIOExecutor.tick(ChunkIOExecutor.java:150) [ChunkIOExecutor.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:673) [MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:387) [ld.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
Caused by: li.cil.repack.com.naef.jnlua.LuaRuntimeException: Invalid kernel.
	at li.cil.oc.server.machine.luac.NativeLuaArchitecture.load(NativeLuaArchitecture.scala:360) ~[NativeLuaArchitecture.class:?]
	... 17 more

Edit: Used foreLuaJ=true, no errors. Computers and robots boot up upon reentering chunk.

--- a/src/main/scala/li/cil/oc/common/tileentity/Robot.scala
+++ b/src/main/scala/li/cil/oc/common/tileentity/Robot.scala
@@ -575,6 +575,7 @@ class Robot extends traits.Computer with traits.PowerInformation with IFluidHand
         for (slot <- componentSlots) {
           getComponentInSlot(slot) match {
             case keyboard: api.internal.Keyboard => buffer.node.connect(keyboard.node)
+            case gpu: li.cil.oc.server.component.GraphicsCard => buffer.node.connect(gpu.node)
             case _ =>
           }
         }

